src/firefox/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "codesy",
+  "title": "codesy.io",
+  "id": "codesy@codesy.io",
+  "description": "codesy is a pay-what-you-want market for the open source community to encourage coders to fix important bugs.",
+  "author": "Luke Crouch",
+  "license": "MPL 2.0",
+  "version": "0.0.9",
+  "main" : "./data/js/index.js",
+  "permissions": {
+    "cross-domain-content": [
+      "https://api.codesy.io/",
+      "https://localhost:8443/"
```

Comment:
Take the `localhost` permission out of the `package.json` so the Firefox add-on only asks for production permission? (Devs will have to add it back to communication with localhost, but we can document that.)